.php_cs.dist

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in('src')
+    ->notPath('Kernel.php')
+;
+
+
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@Symfony' => true,
+        '@PHP71Migration' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'declare_strict_types' => true,
+        'void_return' => true,
+    ])
+    ->setUsingCache(false)
+    ->setRiskyAllowed(true)
```

Comment:
Do we want risky? (Really just asking, this might be the right choice)

Reply:
Actually i don't really know :thinking: It's something made by @vmonjaret

Reply:
It seems to set to true by default 
![image](https://user-images.githubusercontent.com/11029822/48848908-79951400-eda5-11e8-94a2-3beafe796fdf.png)

Reply:
We need to allow risky tests

![image](https://user-images.githubusercontent.com/11029822/48849706-33d94b00-eda7-11e8-80f2-a66a4d24ff32.png)

Reply:
👍